### PR TITLE
Update Docs and Cleanup

### PR DESCRIPTION
**Pull request by endzyme**
Cleaning up the export rootfs functionality in the `in` cmd - also updated docs to better reflect the defaults for users.

### CODE REVIEW

File: assets/in [hunk 1]
```diff
@@ -74,9 +74,6 @@ if [ "$skip_download" = "false" ]; then
     --entrypoint /tmp/print-metadata  \
     "$image_name" > ${destination}/metadata.json
 
-  mkdir -p ${destination}/rootfs/
-  docker export $(cat /tmp/container.cid) | tar -xf - -C ${destination}/rootfs/
```
Comment (vito) [inline assets/in]: This should definitely not go away. This is how `image_resource` works.

File: README.md [hunk 2]
```diff
@@ -83,11 +83,11 @@ The following files will be placed in the destination:
 
 #### Parameters
 
-* `save`: *Optional.* Place a `docker save`d image in the destination.
-* `rootfs`: *Optional.* Place a `.tar` file of the image in the destination.
+* `save`: *Optional.* Place a `docker save`d image in the destination. **DEFAULT: false**
```
Comment (vito) [inline README.md]: Typical formatting for these in our docs is *Optional. Default `false`.*

**Comment by endzyme** [comment]
@vito all set

File: assets/in (hunk 1 above)
Comment (endzyme) [inline assets/in]: Can you point me to the code that uses this? I cannot seem to find it.

Comment (vito) [inline assets/in]: Admittedly it's not documented at all - there's a filesystem schema assumed by `image_resource`. https://github.com/concourse/atc/blob/102e8856983a548797f5c55ebe4f2deea7540ef2/worker/worker.go#L220-L232

Comment (endzyme) [inline assets/in]: I'm unclear as to what is affected by this. Is the ATC worker part of the task running code? If so why is there a hard requirement here and what are the symptoms of removing this image root?

Looks like it's somehow tied into metadata and version? Thought that would all be encapsulated in the contracts of stdin/out.

**Comment by pivotal-issuemaster** [comment]
@endzyme Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/docker-image-resource&pullRequestId=75) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@endzyme Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/docker-image-resource&pullRequestId=75)!

File: assets/in (hunk 1 above)
Comment (endzyme) [inline assets/in]: I'm going to close this as there's no real input here on how I might be able to proceed.

**Comment by endzyme** [comment]
Ok, I'm closing this as there's no real direction on where to go from here.

**Pull request closed without merging**